test/basicTest.js

```diff
@@ -103,6 +104,17 @@ describe("basic tests", function() {
     runCompilerTest(expected, done, undefined, selector);
   }); // it insert into
 
+  it("insert into iframe", function(done) {
+    let selector = "iframe.iframeTarget";
+    styleLoaderOptions.insertInto = selector;
+
+    let expected = 'requiredStyle ';
+
+    runCompilerTest(expected, done, function() {
+      return this.document.querySelector(selector).contentDocument.head.innerHTML;
```

Comment:
`this` is needed. Seems that the test is run in node or so… but scope is explicitly set to window, that's why `this` works.